_adr/0/index.adoc

```diff
@@ -12,39 +12,52 @@ applies_patterns: # What APs does this ADR apply?
 
 include::/_includes/glossary_links.adoc[]
 
-// Top style tips:
-// * Use one sentence per line
-// * No unexpanded acronyms
-// * No undefined jargon - use references to the glossary as appropiate e.g. {glossary-sre}
-
-// No need for a title heading, it's added by the template
+ // The most important goal of an ADR is to let you, your team, new team member or third parties to understand a decision and the context in which it was taken.
+ // It's a https://xkcd.com/1421/[note to yourself] when you will blame the person that made such "terrible" decision.
+ // A second useful aspect is to be the vehicle for discussion on the actual decision while it is being made ; and for non team members to engage into the decisions if they wish to.
+ // 
+ // So assume you reader is not an expert in the problem. Keep it succinct to help readability, feedback received while writing it will tend to make it bigger so start small.
```

Comment:
```suggestion
 // So assume your reader is not an expert in the problem.
 // Keep it succinct to help readability, feedback received while writing it will tend to make it bigger so start small.
```